src/LiveDevelopment/transports/node/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "brackets-livedev-server",
+  "dependencies": {
+    "ws": "~0.4.31"
```

Comment:
Yeah, right now we have to run `npm install` inside the node directory, to run this.
I am trying to figure out a way to move this to root package.json as suggested by you.